tests/tests.toml

```diff
@@ -1383,3 +1383,7 @@ run_with_dbg = true
 [[test]]
 filename = "runtime_errors/test_raise_01.py"
 run_with_dbg = true
+
+[[test]]
+filename = "errors/test_optional_list.py"
```

Comment:
We can rename it to `test_optional.py`. You can add this above line 1350 where you have other error tests listed.

Once you rename, make sure to delete the old reference files.